tests/functional/page-objects/token-page.js

```diff
@@ -2,7 +2,20 @@ import { Selector } from 'testcafe'
 
 const PAGE_CLASS = '.CreateToken'
 const pageClass = Selector(PAGE_CLASS)
+const apiTokens = Selector(`${PAGE_CLASS} .BoxToken__table tbody tr`)
```

Comment:
Should we maybe add a more semantic class for individual tokens?

Reply:
`.BoxToken__table .token`?

Reply:
Just so its not tied to the strucutre of the table.

Reply:
I agree with that reasoning. @wzalazar asked for a BEM formatted classname in the last PR so I'm guessing he'd want to see that here as well. I'm not too familiar with BEM so I'm not sure exactly what the name would look like.

Reply:
ok, that works for me.